io.catenax.week_based_capacity_group/RELEASE_NOTES.md

```diff
@@ -1,9 +1,27 @@
 # Changelog
 All notable changes to this model will be documented in this file.
 
-## [1.0.0]
-- initial version of the aspect model for week-based capacity groups
+## [2.0.0] - 2023-09-18
+
+### Added
+- linkedCapacityGroups added as optional property in aspect model header
+- added loadFactor as optional property in LinkedDemandSeries entity
+- added B7 and SEC to the enum list of unit of measure for load information
```

Comment:
This line is not correct anymore, with the switch to the ext-quantity shared aspect. It would be great for implementers to have a list of UoMs that have been added in the external list, compared to the previous one (with any speaking name that allows implementers that already implemented v1.0.0 of the model to identify the additional UoMs they need to add to their systems). I know that the format in transit changes completely with the switch (from common code to curie), making more changes necessary for implementations then just adding new UoMs, this extra information on the non-technical level would be really helpful as well.

Reply:
A list of UoMs can be found [here](https://confluence.catena-x.net/pages/viewpage.action?pageId=100934075#CX0000DCMStandardv.2.0.0(mergeddocuments)-TABLES) in the new standard.
The last four UoMs were added since 1.0.1. Release_Notes will be updated.